components/match2/wikis/clashroyale/big_match.lua

```diff
@@ -0,0 +1,682 @@
+---
+-- @Liquipedia
+-- wiki=clashroyale
+-- page=Module:BigMatch
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Abbreviation = require('Module:Abbreviation')
+local Arguments = require('Module:Arguments')
+local CardIcon = require('Module:CardIcon')
+local Class = require('Module:Class')
+local Countdown = require('Module:Countdown')
+local DivTable = require('Module:DivTable')
+local Json = require('Module:Json')
+local Links = require('Module:Links')
+local Logic = require('Module:Logic')
+local Namespace = require('Module:Namespace')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Tabs = require('Module:Tabs')
+local VodLink = require('Module:VodLink')
+
+local OpponentDisplay = require('Module:OpponentLibraries').OpponentDisplay
+
+local UTC = '<abbr data-tz="+0:00" title="Coordinated Universal Time (UTC)">UTC</abbr>'
+local NUM_CARDS_PER_PLAYER = 8
+local DEFAULT_CARD = 'transparent'
+local GREEN_CHECK = '[[File:GreenCheck.png|14x14px|link=]]'
+local NO_CHECK = '[[File:NoCheck.png|link=]]'
+local CARD_COLOR_1 = 'blue'
+local CARD_COLOR_2 = 'red'
+
+local LINK_DATA = {
+	preview = {icon = 'File:Preview Icon32.png', text = 'Preview'},
+	lrthread = {icon = 'File:LiveReport32.png', text = 'Live Report Thread'},
+	interview = {icon = 'File:Interview32.png', text = 'Interview'},
+	recap = {icon = 'File:Reviews32.png', text = 'Recap'},
+	vod = {icon = 'File:VOD Icon.png', text = 'Watch VOD'},
+}
+LINK_DATA.review = LINK_DATA.recap
+LINK_DATA.preview2 = LINK_DATA.preview
+LINK_DATA.interview2 = LINK_DATA.interview
```

Comment:
would it be a good idea to let MatchSummary export these and then just require that here?
(would for the future make it so that if matchsummary gets new icons here they are insta supported too)